naughty/arch/3503-qemu-netdev_add-command-failed-2

```diff
@@ -0,0 +1,4 @@
+  File "/work/bots/make-checkout-workdir/test/check-machines-nics", line *, in create
```

Comment:
Please let's not hardcode these internal paths. Just write

    File "test/check-machines-nics", line *, in create

test-policy is clever enough to deal with these.